src/parser/mod.rs

```diff
@@ -7274,6 +7274,16 @@ impl<'a> Parser<'a> {
                 let placeholder = tok.to_string() + &ident.value;
                 Ok(Value::Placeholder(placeholder))
             }
+            tok @ Token::Minus => {
```

Comment:
Should handle `+` as well? And for the sign number value, [another place](https://github.com/sqlparser-rs/sqlparser-rs/blob/4875dadbf5e691dd1e8ef179145829d8ca3ef4d8/src/parser/mod.rs#L1160) represents it with `UnaryOp`, guess should keep consistent? cc @iffyio

Reply:
Oh, what I mean is if we should return the negative number as the UnaryOp expression like the previous implementation.